tabbycat/api/serializers.py

```diff
@@ -398,6 +404,11 @@ def __init__(self, *args, **kwargs):
             if not t.pref('public_break_categories'):
                 self.fields.pop('break_categories')
 
+    def validate(self, data):
+        if data['use_institution_prefix'] and data['institution'] is None:
+            raise serializers.ValidationError("Cannot include institution prefix without institution.")
+        return super().validate(data)
+
```

Comment:
There's a `clean()` method of most models, including `Team.clean()` and `Round.clean()`. Does the REST API library have a means of leveraging this? Would using it help with DRY?

Reply:
I'll check again, but I think I tried and there was some compatibility problem between DRF's ValidationError and Django's. I do like my `==` solution for XNOR logic ;)